docs/getting-started/index.mdx

```diff
@@ -42,179 +56,113 @@ curl -sSL https://ngrok-agent.s3.amazonaws.com/ngrok.asc \
+node service.js
 ```
 
-:::tip
-This example uses `dXNlcjpwYXNzd29yZDE=`, which is a [base64 encoded](https://en.wikipedia.org/wiki/Base64) version of `user:password1`.
-:::
+## 5. Put your app online
 
-You should get the following response:
+Start the ngrok agent by running the following command (replace `8080` if your app is running on a different port).
+The optional `--url` flag connects your app to your reserved domain from step 1—replace `$YOUR_DOMAIN` with this.
 
-```http
-HTTP/2 200 OK
-content-type: text/html
-Successfully Authenticated!
-...
+```bash
+ngrok http 8080 --url $YOUR_DOMAIN
 ```
 
-If you send the same request without the `Authorization` header, you should receive a `401 Unauthorized` response.
+The agent should print a console UI to your terminal to confirm that it's online and forwarding as intended.
+Open the forwarding URL in your browser to see your web application, which is now available over HTTPS with a valid certificate that ngrok automatically manages for you.
 
-</SecureYourApp>
+## 6. Secure your app
 
-ngrok supports many forms of authentication including:
+ngrok makes it simple to add authentication to your app by implementing a [Traffic Policy](/traffic-policy/).
+Using the Agent CLI, you can add a traffic policy to your endpoint by editing your ngrok config file, which was automatically created when you provided your auth token in step 3.
 
-- [OAuth](/traffic-policy/actions/oauth/) (what we just used)
-- [Basic Auth](/traffic-policy/actions/basic-auth/) (what we just used)
-- [IP Restrictions](/traffic-policy/actions/restrict-ips/)
-- [Webhook Verification](/traffic-policy/actions/verify-webhook/)
-- [Mutual TLS](/traffic-policy/actions/terminate-tls/)
-- [OpenID Connect](/traffic-policy/actions/oidc/)
-- [JWT Validation](/traffic-policy/actions/jwt-validation/)
-- [SAML](/traffic-policy/actions/saml/)
+Terminate the agent endpoint that you started up in step 5, then run the following command in your terminal to open `ngrok.yml`:
 
-## Running ngrok Persistently
+```bash
+ngrok config edit
+```
+
+You should see a new console UI with your config version and auth token already set.
+Below the auth token, paste the following snippet into the editor and then save and exit the config file.
+This policy states that whenever an HTTP/S request is made to `$YOUR_DOMAIN`, the agent endpoint should redirect the user to Google OAuth for authentication before proceeding to the app running on port `8080`.
+
+```yaml title="ngrok.yml"
+endpoints:
+  - name: cli-quickstart
+    url: $YOUR_DOMAIN
+    traffic_policy:
+      on_http_request:
+        - actions:
+          - type: oauth
+            config:
+              provider: google
+    upstream:
+      url: 8080
+      protocol: http1
+```
 
-While this guide gets you started with ngrok, you might want your tunnel to keep running even after you close your terminal window. Here are a few options to achieve this:
+Now you can start up your endpoint using its name:
```

Comment:
```suggestion
:::tip
This example uses ngrok's default Google OAuth application. To use your own, see [the OAuth Traffic Policy Action documentation](https://ngrok-docs-git-sg-doc-326-agent-endpoints-cli-f7e245-ngrok-dev.vercel.app/docs/traffic-policy/actions/oauth/#google-example).
:::

Now you can start up your endpoint using its name:
```

Important to note that this example uses ngrok's google oauth config. If you want to config your own google oauth application to be used, gotta do some extra steps. Broadly speaking the ngrok google oauth app isn't useful for production use cases so people will want to know how to configure their own